cinthia_argote/tic_tac_toe/src/BoardTest.java

```diff
@@ -0,0 +1,116 @@
+import static org.junit.Assert.*;
```

Comment:
It is just a suggestion to just import what you are using... I mean just Assert.assertTrue and Assert.assertFalse you don't need to import all Assert.